api/routes/productRouter.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const { productController } = require('../controllers');
+
+const router = express.Router();
+
+router.get('/getCategory', productController.getCategory)
```

Comment:
이미 `HTTP Method(GET)`에 의미가 충분히 담겨 있기 때문에 endpoint 주소에 `get`을 작성하실 필요는 없을 것 같습니다. 
```suggestion
router.get('/categories', productController.getCategory)
```

다른 엔드포인트들도 수정해 주세요~